integration/cypress/integration/devices/list.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { generateNewURL } from "@maas-ui/maas-ui-shared";
+
+import { login } from "../utils";
+
+context("Pools list", () => {
```

Comment:
Both files have the contents for the other file.

Reply:
Thanks, updated with devices removed, and the files no longer transposed 🤦‍♂️